src/NuGet.Core/NuGet.Configuration/PackageNamespaces/ConfigNameSpaceLookup.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+
+namespace NuGet.Configuration
+{
+    public class ConfigNameSpaceLookup
+    {
+        public bool PrefixMatch { get; }
+
+        public HashSet<string> PackageSources { get; }
+
+        public ConfigNameSpaceLookup(bool prefixMatch, HashSet<string> packageSources)
```

Comment:
I wasn't sure if we decided to keep or remote the prefix match here.

Reply:
I can remove `bool prefixMatch`, but need to keep `HashSet<string> packageSources`.
How that sound for you?